README.md

```diff
@@ -47,14 +47,20 @@ is only accessible from the CUL network.
 ```bash
 pipenv install
 FLASK_APP=app.py FLASK_DEBUG=1 ELASTICSEARCH_HOST=127.0.0.1 pipenv run python create_index.py
-FLASK_APP=app.py FLASK_DEBUG=1 ELASTICSEARCH_HOST=127.0.0.1 pipenv run python populate_test_metadata.py
+FLASK_APP=app.py FLASK_DEBUG=1 ELASTICSEARCH_HOST=127.0.0.1 pipenv run python bulk_index.py
 ```
 
-``populate_test_metadata.py`` without parameters populate the index with the
+``bulk_index.py`` without parameters populate the index with the
 list of papers defined in ``tests/data/sample.json``. It take several minutes
 to run. Individual paper IDs may be specified with the ``--paper_id``
 parameter.
 
+To check for missing records, use ``audit.py``:
+
+```bash
+ELASTICSEARCH_HOST=127.0.0.1 pipenv run python audit.py -l list_of_papers.txt -o missing.txt
```

Comment:
It might be good to include ELASTICSEARCH_INDEX in the example so that folks don't accidentally audit against the wrong index.